src/main/java/hw/lesson39/config/AppConfig.java

```diff
@@ -0,0 +1,66 @@
+package hw.lesson39.config;
+
+import hw.lesson39.subtasks.SubTask1;
+import hw.lesson39.subtasks.SubTask2;
+import hw.lesson39.subtasks.SubTask3;
+import hw.lesson39.tasks.Task1;
+import hw.lesson39.tasks.Task2;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+
+@Configuration
+public class AppConfig {
+
+    @Bean
+    @Scope("singleton")
```

Comment:
It's default scope, we can ommit it

Reply:
Updated [53fe7b89](https://github.com/Alexander-Khamitski/TMS_Java_EE/pull/7/commits/53fe7b89546eb2a8c3be81008428891e23c016ce)

Reply:
In other places too

Reply:
Sorry. Didn't check it the first time. Updated: [7e202226](https://github.com/Alexander-Khamitski/TMS_Java_EE/pull/7/commits/7e202226fffa79bf59b2a874b8d1b7ad0d46d576)